playwright/tests/page-objects/block.ts

```diff
@@ -45,8 +45,8 @@ export class Block {
     }
   }
 
-  async select(force: boolean = false) {
-    await this.block.click({ force: force });
+  async select(options: { force?: boolean; position?: { x: number; y: number } } = { force: false }) {
+    await this.block.click({ force: options.force, position: options.position });
```

Comment:
why not simply use the playwright default (which should be the same)
```suggestion
  async select(options?: { force?: boolean; position?: { x: number; y: number } }) {
    await this.block.click(options);
```